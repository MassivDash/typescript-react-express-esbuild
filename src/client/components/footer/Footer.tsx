import React from 'react';

const Footer: React.FunctionComponent = () => {
  return (
    <footer className="bg-gray-200 grid grid-cols-1 lg:grid-cols-5 gap-2">
      <div className="m-3">
        <p>Build and designed with ‚ù§ by Spaceout.pl</p>
        <p>Made with React framework</p>
        <p>Source code at github</p>
      </div>
    </footer>
  );
};

export default Footer;
