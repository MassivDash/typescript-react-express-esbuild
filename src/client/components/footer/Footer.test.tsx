import { render } from '@testing-library/react';
import React from 'react';
import Footer from './Footer';

describe('Footer', () => {
  it('should render', () => {
    const { asFragment, queryByText } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
    expect(queryByText('Made with React framework')).toBeTruthy();
  });
});
