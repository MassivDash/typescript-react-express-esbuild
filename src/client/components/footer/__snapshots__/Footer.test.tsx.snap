// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<DocumentFragment>
  <footer
    class="bg-gray-200 grid grid-cols-1 lg:grid-cols-5 gap-2"
  >
    <div
      class="m-3"
    >
      <p>
        Build and designed with ‚ù§ by Spaceout.pl
      </p>
      <p>
        Made with React framework
      </p>
      <p>
        Source code at github
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
